docs/Configuring_an_Application_Server.md

```diff
@@ -122,3 +122,32 @@ To update the configuration, invoke `curl` as follows.  The first stage builds t
     } | curl -X PUT http://$hs_hostname:8889/irs/<irs-uuid>/service_profiles/<service-profile-uuid>/filter_criteria --data-binary @-
 
 The subscriber will now have the desired configuration. You can confirm this by running the retrieval command again.
+
+## SIP-over-UDP configuration
+
+While it is recommended that you configure your deployment to send SIP over TCP, it is possible to create a SIP-over-UDP deployment, with certain restrictions.
```

Comment:
Maybe worth saying that our default configuration is for SIP-over-TCP - otherwise someone might read the "recommended" bit and ask "so how do I do that?".